pkg/exposer/image.go

```diff
@@ -36,6 +36,8 @@ type inheritedPodInfo struct {
 	volumes        []v1.Volume
 	logLevelArgs   []string
 	logFormatArgs  []string
+	dnsPolicy      v1.DNSPolicy
+	dnsConfig      *v1.PodDNSConfig
```

Comment:
https://github.com/vmware-tanzu/velero/pull/8887
Fixes issue introduced here + the common alias pr.